src/js/common/schemaform/validation.js

```diff
@@ -340,7 +340,7 @@ export function validateFileField(errors, fileList) {
 export function validateBooleanGroup(errors, userGroup, form, schema, errorMessages = {}) {
   const { atLeastOne = 'Please choose at least one option' } = errorMessages;
   const group = userGroup || {};
-  if (!Object.keys(group).filter(item => !!group[item]).length) {
+  if (!Object.keys(group).filter(item => group[item] === true).length) {
```

Comment:
There are other fields in this object that are not booleans. This is sort of lazy, I should probably make the validator take a list of keys.